src/pages/MyProfile.jsx

```diff
@@ -1,15 +1,240 @@
-import React from "react";
+import React, {useEffect, useState} from "react";
+import PopupJS from "reactjs-popup";
 import PageLayout from "../components/PageLayout";
 import Header from "../components/Header";
 import Footer from "../components/Footer";
+import api from "../http/api";
+
+import 'reactjs-popup/dist/index.css';
+import hide from "../assets/hide-pass.svg";
+import show from "../assets/show-pass.svg";
+import {useNavigate, useParams} from "react-router-dom";
+import {toast} from "react-toastify";
 
 function MyProfile() {
+
+    const [profileData, setProfileData] = useState({
+        id: '',
+        username: '',
+        firstName: '',

```

Comment:
you can remove these parameters, if profileData is set to {} after fetching from backend it will fill it the same way.

Reply:
Couldn't get this to work

Reply:
It is not an issue, response should always return these so it should be fine